int32[] label_index
---
string[] labels